AUTHENTICATION.md

```diff
@@ -208,116 +207,214 @@ if (error) {
+
+const auth = new CortiAuth({
+    environment: CortiEnvironment.Eu,
+    tenantName: "YOUR_TENANT_NAME",
 });
 
-app.listen(3000, () => {
-    console.log('Server running on port 3000');
+// Generate authorization URL
+const authUrl = await auth.authorizeURL({
+    clientId: "YOUR_CLIENT_ID",
+    redirectUri: "https://your-app.com/callback",
+    codeChallenge,
 });
+
+// To prevent automatic redirect and get URL only:
+const authUrlNoRedirect = await auth.authorizeURL({
+    clientId: "YOUR_CLIENT_ID",
+    redirectUri: "https://your-app.com/callback",
+    codeChallenge,
+}, { skipRedirect: true });
+```
+
+### Step 2: Handle the Callback and Exchange Code for Tokens
+
+```typescript
+// Extract the authorization code from URL parameters
+const urlParams = new URLSearchParams(window.location.search);
+const code = urlParams.get('code');
+const error = urlParams.get('error');
+
+if (error) {
+    console.error('Authorization failed:', error);
+    return;
+}
+
+if (code) {
+    // Retrieve codeVerifier from SDK if you used authorizePkceUrl (automatically stored)
+    // Otherwise, you need to pass it manually if you generated it yourself
+    const codeVerifier = auth.getCodeVerifier()
+    
+    if (!codeVerifier) {
+        console.error('Code verifier not found');
+        return;
+    }
+    
+    // Exchange the authorization code for tokens using SDK (client-side)
+    const tokenResponse = await auth.getPkceFlowToken({
+        clientId: "YOUR_CLIENT_ID",
+        redirectUri: "https://your-app.com/callback",
+        code: code,
+        codeVerifier: codeVerifier,
+    });
+
+    // Store tokens securely
+    console.log("Token Response: ", tokenResponse)
+}
 ```
 
 ### Step 3: Use the Tokens
 
-Once you have the tokens from your backend server, you can create a `CortiClient` instance:
+Once you have the tokens, you can create a `CortiClient` instance:
 
 ```typescript
-// Frontend: Create CortiClient with tokens from server
+// Create CortiClient with tokens
 const client = new CortiClient({
+    environment: CortiEnvironment.Eu,
+    tenantName: "YOUR_TENANT_NAME",
     auth: {
-        ...token, // Spread the token object received from /api/auth/callback
+        accessToken: accessToken,
+        refreshToken: refreshToken,
         refreshAccessToken: async (refreshToken: string) => {
-            // Make API call to your backend to refresh the token
-            const response = await fetch('/api/auth/refresh', {
-                method: 'POST',
-                headers: {
-                    'Content-Type': 'application/json',
-                },
-                body: JSON.stringify({ refreshToken: refreshToken }),
+            // Refresh tokens using SDK directly (client-side)
+            const refreshResponse = await auth.refreshToken({
+                clientId: "YOUR_CLIENT_ID",
+                clientSecret: '', // PKCE flow doesn't require client secret
```

Comment:
Does refresh token works properly with that? Do we want to add proper types instead of passing empty string?